test/modules/DirectMethodSender/ReporterClients/TestResultReporterClient.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DirectMethodSender
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    internal sealed class TestResultReporterClient : ReporterClientBase
+    {
+        ILogger logger;
+        Uri baseUri;
+        TestResultReportingClient testResultReportingClient = null;
+        TestResultReportingClient TestResultReportingClient
```

Comment:
why do you need this? and where uses it?

Reply:
`ReportStatusAsync()`